tf2onnx/onnx_opset/tensor.py

```diff
@@ -637,6 +637,161 @@ def version_4(cls, ctx, node, **kwargs):
             ctx.copy_shape(node.output[0], cast_node.output[0])
             nodes.append(cast_node)
 
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+        # T output = Slice(T input, Index begin, Index end, Index strides
+        #                 @int begin_mask, @int end_mask, @int ellipsis_mask
+        #                 @int shrink_axis_mask, @int new_axis_mask)
+        # T output = Slice(T input, Tind starts, Tind ends, Tind axes, Tind steps)
+        # "ends" are exclusive, "axes" and "steps" are optional, their default val are [0, ...] and 1
+        begin = node.inputs[1]
+        end = node.inputs[2]
+        strides = node.inputs[3]
+        strides_val = strides.get_tensor_value()
+        if begin.is_const() and end.is_const() and strides.is_const() \
+                and np.all(strides_val == 1):
+            cls.version_4(ctx, node, **kwargs)
+
+        not_supported_attr = ["new_axis_mask"]
+        for attr_name in not_supported_attr:
+            attr = node.get_attr(attr_name)
+            if attr is not None and attr.i != 0:
+                raise ValueError("StridedSlice: attribute " + attr_name + " not supported")
+        onnx_dtype = ctx.get_dtype(node.input[1])
```

Comment:
put these two lines close to the line(line 716) where it used, so we don't need to jump too much to look at its meaning?

Reply:
sure